src/main/java/org/jenkinsci/plugins/workflow/multibranch/ReadTrustedStep.java

```diff
@@ -170,7 +170,7 @@ public String getPath() {
                 if (tip == null) {
                     throw new AbortException("Could not determine exact tip revision of " + branch.getName());
                 }
-                build.addAction(new SCMRevisionAction(tip));
+                build.addAction(new SCMRevisionAction(scmSource, tip));
```

Comment:
https://github.com/jenkinsci/scm-api-plugin/pull/37#discussion_r119865176 :sigh:

Reply:
Adding an extra :sigh: about that for good measure.